### DISCUSSION TITLE
Nested routes for content

### DISCUSSION THREAD

**Discussion by zacharygoulet** [nuxt/content#2081] [Q&A] [answered]
Hi,

I have a route called /blog that is linked to from a menu bar on my site. I use the nuxt css class .router-link-active to style that link when currently on the /blog route or any of its children.

From my understanding, this is possible only when routes are [nested routes](https://nuxt.com/docs/guide/directory-structure/pages/#nested-routes).

However, I couldn't make it work when using in combination with nuxt content's generated routes from content directory.

With this file structure, I manage to get the behavior I want when going to /blog/test,  but not when going to /blog/1mock or /blog/2mock

![image](https://github.com/nuxt/content/assets/92271967/9dd58029-e3e0-492c-928d-b2a98964d3d5)

![image](https://github.com/nuxt/content/assets/92271967/eaf288b0-7174-4773-b0fd-0111b76b924c)

![image](https://github.com/nuxt/content/assets/92271967/b5fcac14-6359-4e55-b3ed-1aa8b7e4916c)

Is there a way to achieve what I'd like to do?

Thanks in advance!

**Comment by nobkd** [comment]
It was done on <https://content.nuxtjs.org> which uses the [`docus` theme](https://github.com/nuxt-themes/docus), so you might want to look there.

![image](https://github.com/nuxt/content/assets/44443899/70c0e3cd-bb44-4e46-8091-c6e2e9a21740)

**Reply by nobkd** [reply, marked as answer]
See https://github.com/nuxt-themes/docus/blob/main/components/app/AppHeaderNavigation.vue#L18-L32 for hints how you *could* do it.
